style.css

```diff
@@ -0,0 +1,106 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@400;500;700;900&display=swap");
+
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+  scroll-behavior: smooth;
+  font-family: "Roboto", sans-serif;
+}
+
+header {
+  background: #3c3a39;
+}
+
+body {
+  overflow-x: hidden;
+  max-width: 375px;
```

Comment:
[Optional]

A very important project requirement was not followed: 
Here at Microverse, we learn to build responsive apps with the use of FlexBox and Grid.

I recommend that you remove the fixed widths in your app and apply the rules of FlexBox to lay out the elements on the UI.
The app at the moment is not responsive. Think about the users that will want to use your app on mobile devices.
https://www.loom.com/share/a61505cea47c4a03b0db7f25409a828f
The app, at the moment, is good to use only by users with devices with 375px.
In accordance with the project requirements, the app should be responsive starting with 375px to 768px.

Printscreen from the project requirements:
![image](https://user-images.githubusercontent.com/83596135/219803011-fd167f18-cf3f-4273-9a06-118267bbac9e.png)

Is only optional because the first reviewer, @aamirkhan2478  missed requiring this from you.